tutorials/getting_started/item_widgets.md

```diff
@@ -0,0 +1,132 @@
+---
+layout: documentation
+title: Pages - Item Widgets
+---
+
+# Custom Default Item Widgets
+
+In addition to customizing the Overview Page and the individual cards as discussed on the previous page, the way that the individual Point Items are shown, or whether or not they are shown at all, can be customized as well.
+In general the steps will include navigating to the Item in the Model tree or the Items Settings, clicking on "Add Metadata", and selecting "Default List Widget" to customize how the Item appears in the automatically generated cards on the Overview Page.
```

Comment:
Those unicode things don't seem to work. We should keep it short and simple.

![Uploading image.png…]()

Reply:
I'm not sure what this comment is referring to. I'm not suing any unicode things, at least not on purpose.

Reply:
See

![image](https://user-images.githubusercontent.com/7161177/111674508-5f0bc200-881c-11eb-9772-781c9feaee85.png)

and 

![image](https://user-images.githubusercontent.com/7161177/111679723-be200580-8821-11eb-87c5-bd8ced2c55cd.png)

Reply:
Weird. That's not showing up in VSCode or vim on my machine. I retyped the sentence in this latest push. Hopefully they are gone now.

Reply:
Yep they are gone. Thanks. but you introduced one typo.
I will fix it myself and then merge. 🙂